profiles/seccomp/default.json

```diff
@@ -352,6 +352,7 @@
 				"statfs",
 				"statfs64",
 				"statx",
+				"swapcontext",
```

Comment:
This JSON file is generated from the Go source in https://github.com/moby/moby/blob/master/profiles/seccomp/default_linux.go, so that file would need to be updated as well.

If this syscall is only used on ppc, it should probably be in a conditional rule, similar to (or perhaps as part of) https://github.com/moby/moby/blob/686be57d0a6e514c0cddb2f3ac9cbb3cbef87f5f/profiles/seccomp/default_linux.go#L473-L483

Also looks like there's at least one test that needs to be adjusted;

```
=== RUN   TestUnmarshalDefaultProfile
    seccomp_test.go:196: assertion failed: 
        --- expected.Syscalls
        +++ profile.Syscalls
        {[]*seccomp.Syscall}[0].LinuxSyscall.Names[?->299]:
        	-: <non-existent>
        	+: "swapcontext"
```

Reply:
Fixed this, anything else? (: